### PR TITLE
Problem: outdated GitLab instructions on GitHub page and check sourcetree recommended on windows

### PR DESCRIPTION
**Pull request by dhruvakumar26**
Solution: As we already switched from GitLab, the instructions mentioning GitLab are removed.
 Sourcetree is a simple and user friendly tool with transparent navigation and a bunch of useful features . so, it is still recommended for windows users.

### CODE REVIEW

**Comment by gabm** [comment]
thank you for working on this @dhruvakumar26 ... could you also revise the following sections?

- https://tuc-nt-df.github.io/student-guide/content-ops/how-to-contribute-to-this-guide/#the-repository-for-this-guide
- https://tuc-nt-df.github.io/student-guide/content-onboarding/working-with-GitHub/#provide-your-real-full-name-and-create-an-avatar
- https://tuc-nt-df.github.io/student-guide/content-onboarding/onboarding/#additional-steps
- https://tuc-nt-df.github.io/student-guide/content-onboarding/onboarding/#prepare-for-software-version-control
- https://tuc-nt-df.github.io/student-guide/content-knowledgebase/linux_hints_for_windows_users/#getting-gitlab-access-working


Most of them need fixing of gitlab-specific links... Thanks!

File: docs/content-onboarding/onboarding.md [hunk 1]
```diff
@@ -129,13 +132,13 @@ If the welcome message doesn’t appear, run SSH’s verbose mode by replacing `
 
 ## Additional Steps
 * Install Miniconda \(based on Python 3\) by following the [Conda How-To](https://draive.com/link_dev/guide/01_Conda_Setup/) and install the most important dependencies.
-* [Create an avatar](http://avatarmaker.com/) for your GitLab account. Add it to your GitLab [profile](https://gitlab.com/-/profile).
+* [Create an avatar](http://avatarmaker.com/) for your GitHub account. Add it to your GitHub [profile](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2F-%2Fprofile).
```
Comment (gabm) [inline docs/content-onboarding/onboarding.md:135]: I think that link is not correct... an you check?

**Review by gabm — CHANGES_REQUESTED** [review]

File: docs/content-onboarding/onboarding.md (hunk 1 above)
Comment (dhruvakumar26) [reply docs/content-onboarding/onboarding.md]: I have updated the GitHub profile link to https://github.com/settings/profile 
👍

**Review by dhruvakumar26 — COMMENTED** [review]
Updated the GitHub profile link to https://github.com/settings/profile

**Pull request merged**